.github/workflows/ci.yaml

```
@@ -0,0 +1,10 @@
+name: ci
+on: [push, pull_request]
```

```suggestion
on:
  push:
    branches:
    - master
  pull_request: {}
```

I find this works much better.